configuration/jsr223.md

```diff
@@ -118,23 +118,24 @@ Scripts should be placed in the `${OPENHAB_CONF}/automation/jsr223/` directory.
 This directory will vary, [based on the type of openHAB installation used](https://www.openhab.org/docs/installation/linux.html#installation).
 For example, Linux installations created with a package installer will use `/etc/openhab/automation/jsr223/`, and manual installations will use `/opt/openhab/conf/automation/jsr223/`.
 
-When openHAB starts, scripts will be loaded in an order based on their file name.
-If the scripts have the same name, which should rarely happen, the parent directories will be considered in the sort.
+When openHAB starts, scripts are loaded at start level 40 by default (in no particular order). The start level for each script can be overriden by specifying a start level either in the filename (`./my_script.sl50.py`) or containing directory (`./sl50/my_script.py`). The runtime provides no explicit dependency mechanism or ordering, yet scripts are loaded one at a time so can be ordered via start level if desired.
```

Comment:
Please use one line per sentence.
Website build will take care of proper paragraphs and presentation on the website.